src/System.Text.RegularExpressions/src/Properties/AssemblyInfo.cs

```diff
@@ -29,3 +30,4 @@
 [assembly: ComVisible(false)]
 [assembly: CLSCompliant(true)]
 
+[assembly: InternalsVisibleTo("System.Text.RegularExpressions.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
```

Comment:
We generally would like to avoid exposing internals to our test assemblies if we can help it. For this test does it matter if it is exactly the reference equivalent of of _emptyGroup? Why not just check to see that things are empty.